- Description: kernel: makefile problem.
  Symptom:     Make clean fails.
  Problem:     The makefile in arch/s390 contains a rule for the
               non-existent oprofile subfolder. Because of this
               make clean fails.
  Solution:    Remove oprofile rul from arch/s390/Makefile.
  Problem-ID:  -
  Archs:       s390-31, s390-64

--- linux-2.5/arch/s390/Makefile	21 Jun 2004 17:08:54 -0000	1.41.2.2
+++ linux-2.5/arch/s390/Makefile	24 Jun 2004 12:08:04 -0000	1.41.2.3
@@ -50,9 +50,6 @@
 drivers-y	+= drivers/s390/
 drivers-$(CONFIG_MATHEMU) += arch/$(ARCH)/math-emu/
 
-# must be linked after kernel
-drivers-$(CONFIG_OPROFILE)	+= arch/s390/oprofile/
-
 boot		:= arch/$(ARCH)/boot
 
 all: image kerntypes.o

